### DISCUSSION TITLE
Pass command-Line options in a single instance

### DISCUSSION THREAD

**Discussion by ftk789** [Chatterino/chatterino2#3567] [Ideas]
For an example if i do Chatterino.exe -c ftk789 , instead of running a new instance, would be Chatterino.exe -singleInstance -c ftk789, and opens the channel in the already running instance

**Comment by Mm2PL** [comment]
Would need something like https://github.com/Chatterino/chatterino2/issues/317

**Reply by ftk789** [reply]
Aww man, Would've been good to have an IPC then, Thank you for your reply

**Reply by Mm2PL** [reply]
It's still something that is doable. I wouldn't doubt that the topic of IPC comes back again.

**Reply by ftk789** [reply]
I would love it to be a thing, since my friends use chatterino a lot, i wanted to make a script that uses the IPC or instances communications, but yea... Hopefully that's a thing

**Comment by leon-richardt** [comment]
For reference, https://github.com/streamlink/streamlink-twitch-gui/issues/607 would also like this to be a thing
